Description: Correct pod NAME issue in LedgerSMB/DBObject/EOY.pm
 Has a colon instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472565&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/EOY.pm
+++ b/LedgerSMB/DBObject/EOY.pm
@@ -1,6 +1,6 @@
 =head1 NAME
 
-LedgerSMB::DBObject::EOY: End of Year handling module
+LedgerSMB::DBObject::EOY - End of Year handling module
 
 =head1 SYNOPSYS
 
